services/ui-src/src/components/forms/AdminBannerForm.tsx

```diff
@@ -7,18 +7,31 @@ import { bannerId } from "../../constants";
 import { REPLACE_BANNER_FAILED } from "verbiage/errors";
 // data
 import { form as formJson } from "verbiage/forms/adminBannerForm";
+import { calculateTimeByDateType, convertDateEtToUtc } from "utils";
 
 export const AdminBannerForm = ({ writeAdminBanner, ...props }: Props) => {
   const [error, setError] = useState<string>();
 
+  const convertTime = (date: string, fieldName: string) => {
+    const year = parseInt(date.split("/")?.[2]);
+    const month = parseInt(date.split("/")?.[0]);
+    const day = parseInt(date.split("/")?.[1]);
+    let convertedTime = undefined;
+    if (year && month && day) {
+      const time = calculateTimeByDateType(fieldName);
+      convertedTime = convertDateEtToUtc({ year, month, day }, time);
+    }
+    return convertedTime;
+  };
```

Comment:
Cool. Let's move this whole thing to `utils/time` with the others and import it for reference since we'll be using it in lots of places.

Reply:
Nit: Change name to match other time utils, like `convertDatetimeStringToNumber`